<?xml version="1.0"?>
<launch>
  <arg name="output"                    default="screen"/>
  <arg name="launch_prefix"             default="'nice -n -10'" />
  <arg name="standalone"                default="true" />
  <!-- setting sampling rate to 0.0 means that the sampling rate is derived from the FT filter configuration of the device -->
  <arg name="time_step"                 default="0.0025"/>
  <arg name="num_spinners"              default="1" />
  <arg name="rokubimini_setup_file"     default="$(find rokubimini_cosmo)/config/setup.yaml" />
  <arg name="sensor_config_name"        default="" />  
  <arg name="ros_publishers_file"       default="$(find rokubimini_cosmo)/config/publishers.yaml"/>
  <arg name="node_path"                 default="$(find rokubimini_cosmo)/../../devel/lib/rokubimini_cosmo/rokubimini_cosmo_node"/>
  <arg name="password"                  default=""/>

  <!-- Launch the lowlevel controller: Setcap and ROS Node -->
  <node pkg="rokubimini_cosmo" type="rokubimini_launch.sh" name="rokubimini_launch" output="$(arg output)"
        args="$(arg launch_prefix)
              $(arg standalone)
              $(arg time_step)
              $(arg num_spinners)
              $(arg rokubimini_setup_file)
              $(arg sensor_config_name)
              $(arg ros_publishers_file)
              $(arg node_path)
              $(arg password)">
</node>
</launch>
